### ISSUE TITLE
Question for "scale_loss_shift" parameter

### ISSUE THREAD

**Issue by computervisionta** [TuSimple/simpledet#16]
Thanks for the good work. I have a question for "scale_loss_shift' parameter, I found this parameter is used in the following:
scale_loss_shift = 128.0 if p.fp16 else 1.0
        cls_loss = mx.sym.SoftmaxOutput(data=cls_logit_reshape,
                                        label=cls_label,
                                        multi_output=True,
                                        normalization='valid',
                                        use_ignore=True,
                                        ignore_label=-1,
                                        grad_scale=1.0*scale_loss_shift,
                                        name='rpn_cls_loss')

question: why fp16 or fp32 can have an impact on grad_scale of softmax loss? Any advice will be appreciated, thanks

**Comment by computervisionta** [comment]
Got it! Thanks
